ApplicationCode/GeoMech/GeoMechDataModel/RigFemPartResultCalculatorMudWeightWindow.cpp

```diff
@@ -389,7 +394,7 @@ RigFemScalarResultFrames* RigFemPartResultCalculatorMudWeightWindow::calculate(
             size_t kMin       = std::min( k, kRefLayer );
             size_t kMax       = std::max( k, kRefLayer );
 
-            if ( elmType == HEX8P && validIndex )
+            if ( ( elmType == HEX8P || elmType == HEX8 ) && validIndex )
```

Comment:
This check is done many times in the code. Suggest moving this check into RgFemTypes

Reply:
Done.